scripts/build/config.js

```diff
@@ -24,7 +24,7 @@ const PROJECT_ROOT = path.resolve(__dirname, "../..");
 /** @type {Bundle[]} */
 const parsers = [
   {
-    input: "src/language-js/parser-babylon.js",
+    input: "src/language-js/parser-babel.js",
```

Comment:
This is a breaking change for `standalone` users so I'd expect the file renaming to be happened in v2, we should leave the filename as-is for now.

Reply:
Or we could include both names in the package and print a warning if the `babylon` file is used.

Reply:
The size will be doubled for babylon, I'm not sure if it's worth.

Reply:
Ok Ill leave it as is for now. And leave a note that it should be updated in the next major release